### PR TITLE
Bump bloom to 1.0.0 and add generic router [DAH-746]

### PR DESCRIPTION
**Pull request by james-wills-sf**
Tangent from [DAH-746]

This PR does the following:
- remove `@sf-digital-services` dependency, we don't need it anymore
- Add back `@bloom-housing` dependency and bump it to the 1.0.0 release
- Add our SF routing solution in the form of `<NavigationProvider />`
  - This implements an interface for routing. Right now it's very simple but can be updated to work with react router or whatever we want.
- Update all existing usages of `<a />` and replace them with `<Link />`. We should use `<Link />` everywhere from now on.

[DAH-746]: https://sfgovdt.jira.com/browse/DAH-746

### CODE REVIEW

File: app/javascript/components/MetaTags.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import Head from "next/head"
+
+export interface MetaTagsProps {
+  title: string
+  image?: string
+  description: string
+}
+
+const MetaTags = (props: MetaTagsProps) => (
```
Comment (james-wills-sf) [inline app/javascript/components/MetaTags.tsx:11]: After bumping to 1.0.0 it looked like this Metatags component was removed from ui-components, does that sound right, @software-project?

I just copied this code directly from the public reference app.

File: app/javascript/util/routeUtil.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ const getLocalizedPath = (
 }
 
 const localizedPathGetter = (newPathNonLocalized: string) => (
-  currentPath: string | undefined
+  currentPath: string | undefined = window.location.pathname
```
Comment (james-wills-sf) [inline app/javascript/util/routeUtil.ts:32]: I just added this default so this code is still testable but we don't need to pass the pathname in explicitly everywhere.

File: app/javascript/components/MetaTags.tsx (hunk 1 above)
Comment (software-project) [reply app/javascript/components/MetaTags.tsx:11]: Thats correct. They were moved to the reference app.

**Review by software-project — APPROVED** [review]
@james-wills-sf nice work. Links are working, also with translations. Nice work on simplyfying stuff as well.

**Pull request merged**